repo/meta/index.json

```diff
@@ -83,6 +83,23 @@
       }
     },
     {
+      "currentVersion":"0.3.16.4",
+      "description":"High Performance Stream Processing with C++ and Apache Mesos",
+      "framework":true,
+      "name":"concord",
+      "selected":false,
+      "tags":[
+        "stream",
+        "concord",
```

Comment:
I don't think `concord` is needed as a tag here since it's already the package name